index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <base href="2017">
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+    <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=no">
```

Comment:
Hmm okay, makes sense. But there has to be another way to prevent zooming when the application is in a web-wrapper.